integration-tests/src/test/java/uk/gov/di/authentication/api/AuthorisationIntegrationTest.java

```diff
@@ -100,7 +100,14 @@ class AuthorisationIntegrationTest extends ApiGatewayHandlerIntegrationTest {
     private static final String AM_CLIENT_ID = "am-test-client";
     private static final String TEST_EMAIL_ADDRESS = "joe.bloggs@digital.cabinet-office.gov.uk";
     private static final String TEST_PASSWORD = "password";
-    private static final KeyPair KEY_PAIR = KeyPairHelper.GENERATE_RSA_KEY_PAIR();
+    private static final KeyPair RP_KEY_PAIR = KeyPairHelper.GENERATE_RSA_KEY_PAIR();
+    private static final KeyPair AUTH_ENCRYPTION_KEY_PAIR = KeyPairHelper.GENERATE_RSA_KEY_PAIR();
+    private static final String AUTH_PUBLIC_ENCRYPTION_KEY =
+            "-----BEGIN PUBLIC KEY-----\n"
+                    + Base64.getMimeEncoder()
+                            .encodeToString(AUTH_ENCRYPTION_KEY_PAIR.getPublic().getEncoded())
+                    + "\n-----END PUBLIC KEY-----\n";
+    private static final KeyPair DCMAW_ENCRYPTION_KEY_PAIR = generateRsaKeyPair();
```

Comment:
such a nitpick but more for my curiosity how come we have two methods of generating key pairs?